Path-Check/covid-safe-paths
# Start/stop location tracking if needed when checking status

This updates the new Main.js tracking code to optionally start and stop the location tracking service when checking status.

#### Linked issues:

Fixes #797

#### Screenshots:

NA

#### How to test:

- Check that the service starts after "All finished" onboarding, then and moving to the main screen.
- Check that the service stops when stopping from app permissions, app settings, and when disabling location tracking at the OS level.
- Check that the service starts up again after toggling in settings.
